### ISSUE TITLE
Questions for Tuning Details

### ISSUE THREAD

**Issue by KangLiao929** [showlab/Show-o#63]
Hi,

Thanks (*with the star*) for your awesome work! Recently I plan to fine-tune the show-o on the specific downstream tasks such as object grounding, in which I mainly focus on generating from the text with object coordinates to image and estimating the coordinates of the mentioned objects (rather than accurate MMU performance). However, I have some questions regarding the tuning details.

1. Should I perform both 'llava_pretrain' and 'llava_tuning' for the above purpose? Will 'llava_tuning' be enough for this specific task?
2. For the datasets, can I replace all datasets used in three stages (t2i, lm, mmu) with my object grounding dataset? Should I keep the original datasets of t2i to eliminate the performance degradation?

Looking forward to your reply and suggestion!

**Comment by Sierkinhane** [comment]
I think you'd better include both (it's fast to perform 'llava_pretrain'). To maintain t2i capability, I would suggest you include high-quality t2i data when performing the tuning process.

**Comment by KangLiao929** [comment]
> I think you'd better include both (it's fast to perform 'llava_pretrain'). To maintain t2i capability, I would suggest you include high-quality t2i data when performing the tuning process.

Thanks for your quick feedback. The size of the targeted downstream dataset is much smaller than the high-quality t2i data, only around 10K images with annotated labels. Would the unified model tend to ignore the specific task during the mixed training with large-scale datasets?

**Comment by Sierkinhane** [comment]
It's like llava tuning, which commonly tunes only one epoch to avoid overfiting. In our experiments, it's quite necessary to balance each kind of data during pre-training and fine-tuning to pursue better performance.
